client/src/pages/Home.js

```diff
@@ -43,24 +85,43 @@ const StyledP = styled.p`
   line-height: 2.1rem;
   letter-spacing: 0.1px;
   margin: 0;
+  
+  @media only screen and (min-width: 600px){
+    //theme.typography.heading.one is not enough size
+    font-size: 1vw;
+    text-align: left;
+    color: white;
+  }
 `;
 
 const OnboardingContainer = styled.div`
   margin-top: 4rem;
+
+  @media only screen and (min-width: 600px){
+    display: "flex";
```

Comment:
Don't think the double quotes are needed?

Reply:
ok